index.html

```diff
@@ -0,0 +1,118 @@
+                    <a href="about.html">About</a>
+                    <a href="">Program</a>
+                    <a href="">Join</a>
+                    <a href="">Sponsor</a>
+                    <a href="">News</a>
+                    <a href="">BC Campaign</a>
+                </div>
+            </div>
+            <div class="nav-wrapper-desktop">
+                <img class="img-home" src="images/bioLogo.png" alt="">
+                <div>
+                    <a href="about.html">About</a>
+                    <a href="">Program</a>
+                    <a href="">Join</a>
+                    <a href="">Sponsor</a>
+                    <a href="">News</a>
+                    <a href="">BC Campaign</a>
+                </div>
+            </div>
+        </nav>
+        <div class="presentation-container">
+            <header>
+                <h3>"Hello! Science world"</h3>
+                <h1>WORLD BIOTECH CONGRESS 2022</h1>
+            </header>
+            <p class="main-description">Lorem ipsum dolor sit amet consectetur, adipisicing elit. Sed, facere? Non blanditiis eligendi eaque adipisci consectetur expedita, nobis quasi tenetur dolore, officia quas fuga ratione debitis dolores, molestias possimus illum.</p>
+            <div>
+                <h2>2022.10.15(THU) ~ 16(FRI)</h2>
+                <p>@ National Museum of Ecuador, Biodimed and more</p>
+            </div>
+        </div>
+    </section>
+    <section class="main-program" id="main-program">
+        <h2>Main Program</h2>
+        <h6 class="line">────</h6>
+        <div class="main-schedule">
+            <article>
+                <img src="images/lecture1.png" alt="">
+                <h4>Lecture</h4>
+                <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Distinctio maiores</p>
+            </article>
+            <article>
+                <img src="images/lecture2.png" alt="">
+                <h4>BC Exhibition</h4>
+                <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Distinctio maiores</p>
+            </article>
+            <article>
+                <img src="images/lecture3.png" alt="">
+                <h4>Forum</h4>
+                <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Distinctio maiores</p>
+            </article>
+            <article>
+                <img src="images/lecture4.png" alt="">
+                <h4>Workshop</h4>
+                <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Distinctio maiores</p>
+            </article>
+            <article>
+                <img src="images/lecture5.png" alt="">
+                <h4>BC Ignite</h4>
+                <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Distinctio maiores</p>
+            </article>
+        </div>
+        <button>Join The web Development Course</button>
+    </section>
+    <section class="partner" id="partner">
+        <h2>Partner</h2>
```

Comment:
- [Optional] Please use an indentation of two spaces for readability in all your documents. This is best practice and also a convention. Check [here](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md) for more details, please.